src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -97,7 +103,9 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
         htmlFor={"chk-kidsMode"}
         className={`checkbox ${profile?.kidsMode && "checkbox-checked"}`}
       >
-        Kids Mode
+        <span className="title">Kids Mode</span>
```

Comment:
Looks like a merge is needed here, I moved these above the logout button & added a mirror video toggle. Let's have the anonymous mode option last of all.